www/js/goals.js

```diff
@@ -2,9 +2,10 @@
 
 angular.module('emission.main.goals',['emission.services', 'emission.plugin.logger',
                 'ngSanitize', 'ngAnimate',
-                'emission.splash.referral', 'angularLocalStorage'])
+                'emission.splash.referral', 'angularLocalStorage',
+                'ng-walkthrough', 'nzTour'])
 
-.controller('GoalsCtrl', function(CommHelper, $state, $ionicLoading, $scope, $rootScope, $ionicModal,
+.controller('GoalsCtrl', function(CommHelper, $state, $ionicLoading, $scope, $rootScope, $ionicModal, nzTour, Config,
```

Comment:
nit: why do we need to add the `Config` service here? Don't seem to use it in the rest of the code....